salt/utils/reactor.py

```diff
@@ -442,15 +450,7 @@ def run(self, low):
             # and kwargs['kwarg'] contain the positional and keyword arguments
             # that will be passed to the client interface to execute the
             # desired runner/wheel/remote-exec/etc. function.
-            ret = l_fun(*args, **kwargs)
-
-            if ret is False:
-                log.error('Reactor \'%s\' failed  to execute %s \'%s\': '
-                            'TaskPool queue is full!'
-                            ' Consider tuning reactor_worker_threads and/or'
-                            ' reactor_worker_hwm',
-                            low['__id__'], low['state'], low['fun']
-                )
+            l_fun(*args, **kwargs)
```

Comment:
Why we're getting to ignore the result here?

Reply:
I guess the question is what should we do with the ret in reactors case? it was previously used to check it the Queue.Queue was full, but otherwise tossed. can we meaningfully do anything that isn't already emitted in the master log?